packages/manager/apps/pci-rancher/src/components/layout-helpers/CreateRancher/CreateRancher.tsx

```diff
@@ -0,0 +1,244 @@
+}
+
+const CreateRancher: React.FC<CreateRancherProps> = ({
+  plans,
+  versions,
+  onCreateRancher,
+  hasRancherCreationError,
+  projectId,
+}) => {
+  const navigate = useNavigate();
+  const { t } = useTranslation([
+    'pci-rancher/dashboard',
+    'pci-rancher/listing',
+  ]);
+  const [rancherName, setRancherName] = useState(null);
+
+  const activeVersion = versions?.filter((v) => v.status === 'AVAILABLE')[0];
+  const activePlan = plans?.filter((v) => v.status === 'AVAILABLE')[0];
+  const isValidName = rancherName !== '' && isValidRancherName(rancherName);
+  const hasInputError = rancherName !== null && !isValidName;
+
+  const onCreateClick = (rancherPayload: CreateRancherPayload) => {
+    onCreateRancher(rancherPayload);
+    navigate(`/pci/projects/${projectId}/rancher`);
+  };
+
+  const onCancelClick = () =>
+    navigate(`/pci/projects/${projectId}/rancher/onboarding`);
+
+  return (
+    <div className="max-w-3xl">
+      <Title>{t('createRancherTitle')}</Title>
+      <OsdsMessage
+        color={ODS_THEME_COLOR_INTENT.info}
+        type={ODS_MESSAGE_TYPE.info}
+        className="my-6"
+      >
+        <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+          {t('createRancherInfoMessage')} <br />
+        </OsdsText>
+      </OsdsMessage>
+      {hasRancherCreationError && (
+        <OsdsMessage
+          color={ODS_THEME_COLOR_INTENT.error}
+          type={ODS_MESSAGE_TYPE.error}
+          className="my-6"
+        >
+          <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+            {t('createRancherError')} <br />
+          </OsdsText>
+        </OsdsMessage>
+      )}
+      <div>
+        <div className="mb-8">
+          <Block>
+            <Subtitle>{t('createRancherName')}</Subtitle>
+          </Block>
+          <Block>
+            <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+              {t('createRancherDescription')}
+            </OsdsText>
+          </Block>
+          <OsdsInput
+            aria-label="rancher-name-input"
+            type={ODS_INPUT_TYPE.text}
+            color={
+              hasInputError
+                ? ODS_THEME_COLOR_INTENT.error
+                : ODS_THEME_COLOR_INTENT.info
+            }
+            className="my-3 w-1/3"
+            value={rancherName}
+            onOdsValueChange={(
+              e: OsdsInputCustomEvent<OdsInputValueChangeEventDetail>,
+            ) => setRancherName(e.target.value as string)}
+          />
+        </div>
+
+        <div className="my-3">
+          <Subtitle>{t('createRancherServiceLevel')}</Subtitle>
+        </div>
+        <Block>
+          <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+            {t('createRancherServiceLevelTitle')}
+          </OsdsText>
+        </Block>
+        <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+          {t('createRancherServiceLevelDescription')}
+        </OsdsText>
+        <div className="flex my-5">
+          {plans?.map((v) => (
+            <TileSection
```

Comment:
You should use the radio + tile like vrackservices
packages/manager/apps/vrack-services/src/pages/create/components/RegionFormField.tsx